clients/src/main/java/org/apache/kafka/clients/admin/AlterPartitionReassignmentsOptions.java

```diff
@@ -28,4 +28,25 @@
  */
 @InterfaceStability.Evolving
 public class AlterPartitionReassignmentsOptions extends AbstractOptions<AlterPartitionReassignmentsOptions> {
+
+    private Boolean allowReplicationFactorChange = true;
```

Comment:
`boolean`

Reply:
This is defined in the KIP, but I'm happy to modify it.